client/src/components/Help/Help.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+// ---| core |---
+import { cn } from 'tools'
+
+// ---| pages |---
+// ---| screens |---
+// ---| components |---
+import Popup, { PopupProps } from 'components/Popup'
+import Button from 'components/Button'
+import Card from 'components/Card'
+import Text from 'components/Text'
+
+
+// ---| self |---
+import css from './Help.module.scss'
+import Messages, { MessageItem } from 'components/Messages'
+
+export type HelpProps = {
+  className?: string
```

Comment:
Optional: I prefer have `className` and `children` in the end of the list of props but before callbacks